src/main/java/io/unlogged/core/processor/UnloggedProcessor.java

```diff
@@ -361,12 +365,26 @@ private Object tryGetFilerField(Class<?> delegateClass, Object instance) {
         }
     }
 
+	public static long getDefaultCounter() {
+		return defaultCounter;
+	}
+
+	public static ExperimentalSDKFlagType getArgumentSend() {
+		return argumentSend;
+	}
+
     @Override
     public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
 
         if (System.getProperty("unlogged.disable", null) != null) return false;
 
 
+		for (Element element : roundEnv.getElementsAnnotatedWith(Unlogged.class)) {
+			Unlogged unlogged = element.getAnnotation(Unlogged.class);
+			defaultCounter = Long.parseLong(unlogged.loggingFrequency());
+			argumentSend = unlogged.argumentSend();
```

Comment:
Because of that call to UnloggedProcessor from MethodVisitor, that methodVisitor cannot be tested without a call to UnloggedProcessor to set the values for argumentSend/defaultCounter. and for that we end up doing mocking for roundEnvironment etc. Got to disconnect that link and pass the two values down.

Reply:
Yes, now we can just mock UnloggedProcessorConfig class to test MethodVisitorWithoutProbe manipulations.